datalad/interface/base.py

```diff
@@ -570,7 +570,13 @@ def setup_parser(cls, parser):
                     cdoc = cdoc[1:-1]
                 help += '  Constraints: %s' % cdoc
             if defaults_idx >= 0:
-                help += " [Default: %r]" % (defaults[defaults_idx],)
+                # if it is a flag, in commandline it makes little sense to show
+                # showing the Default: (likely boolean).
+                #   See https://github.com/datalad/datalad/issues/3203
+                if not parser_kwargs.get('action', '').startswith('store_'):
+                    # [Default: None] also makes little sense for cmdline
```

Comment:
The argument for `[Default: None]` being confusing for command line holds true for non-flag options as well or am I missing something?

Reply:
BTW: Same argument is true for `True` and `False` in general. It's not (necessarily) referring to those strings.

Reply:
hm, ... what if there is a `choice` allowing for bool and some strings, and default is a bool?  Note: This block is executed only for non `store_`, so all flags are excluded already by here, but IMHO it shouldn't be generalized solely based on a default value

Reply:
but now I am getting a better clue on what you meant ;)  indeed it is unlikely we have `bool` specified as a value (e.g. as `True`) in command line... we do though have `assure_bool` etc handling which was/is used by crawler, but I am not sure we have anything like that for command line options.  I would though probably not dive into deep analysis at this point -- could be improved later if we run into such odd places.